packages/hash/api/src/graph/system-user.ts

```diff
@@ -62,14 +64,15 @@ export const ensureSystemUserAccountIdExists = async (params: {
   );
 
   if (existingSystemUserEntity) {
-    systemUserAccountId = extractEntityUuidFromEntityId(
-      existingSystemUserEntity.metadata.editionId.baseId,
+    systemUserAccountId = extractAccountId(
+      brand(existingSystemUserEntity.metadata.editionId.baseId),
```

Comment:
Places like this make me wonder if we want to actually use the casting form rather than the function. Genuinely unsure which, the `brand` is slightly more ergonomic it seems because it infers from the type, but also here it's very unclear to me why you're branding the thing (although I'm pretty sure it's because of my suggestion about having a variant of `EntityId`)

Reply:
Hmm, good point. It is easy to cast something incorrectly, whereas the `as` cast would check the base type. And yes the reason for this is because of the `EntityId` variant

Reply:
Changed in 79a98adc0227c976094cc723a4b84e1362ffa60d